setup.py

```diff
@@ -16,7 +16,8 @@
 install_requires = [
     'six',
     # >= 8 9s because of breaking API change
-    'html5lib>=0.99999999',
+    # the 'pre' suffix is needed for supporting '1.0b*' versions
```

Comment:
As in 0.99999999**pre**. It's there to have pip treat it as a dependency on a prerelease version. Otherwise it won't select prerelease versions like 1.0b9, since it sees "0.99999999" and thinks it's a proper release. [See this for its effects](https://gist.github.com/indivisible/5f70480df776f48907218006278da5af).